tests/accept_tests.py

```diff
@@ -49,3 +50,69 @@ def test_accept_final_comment(self):
         new_id = (set(comments.keys()) - set(self.comments.keys())).pop()
         comment = comments[new_id]['comment']
         self.assertEqual('Project "{}" accepted. The following packages have been submitted to openSUSE:Factory: wine.'.format(self.prj), comment)
+
+    def test_accept_new_multibuild_package(self):
+        wf = self.setup_wf()
+
+        staging = wf.create_staging('A', freeze=True)
+
+        project = wf.create_project('devel:gcc')
+        package = OBSLocal.Package(name='gcc9', project=project)
+        package.create_commit(filename='gcc9.spec')
+        package.create_commit(filename='gcc9-tests.spec')
+        package.create_commit('<multibuild><flavor>gcc9-tests.spec</flavor></multibuild>', filename='_multibuild')
+        wf.submit_package(package)
+
+        ret = SelectCommand(wf.api, staging.name).perform(['gcc9'])
+        ac = AcceptCommand(wf.api)
+        self.assertEqual(True, ac.accept_all(['A'], True))
+
+        # no stale links
+        self.assertEqual([], package_list(wf.apiurl, staging.name))
+        self.assertEqual(['gcc9', 'wine'], package_list(wf.apiurl, wf.project))
+
+    def test_accept_new_multispec_package(self):
+        wf = self.setup_wf()
+
+        staging = wf.create_staging('A', freeze=True)
+
+        project = wf.create_project('devel:gcc')
+        package = OBSLocal.Package(name='gcc9', project=project)
+        package.create_commit(filename='gcc9.spec')
+        package.create_commit(filename='gcc9-tests.spec')
+        wf.submit_package(package)
+
+        ret = SelectCommand(wf.api, staging.name).perform(['gcc9'])
+        ac = AcceptCommand(wf.api)
+        self.assertEqual(True, ac.accept_all(['A'], True))
+
+        # no stale links
+        self.assertEqual([], package_list(wf.apiurl, staging.name))
+        self.assertEqual(['gcc9', 'gcc9-tests', 'wine'], package_list(wf.apiurl, wf.project))
+
+    def test_accept_switch_to_multibuild_package(self):
```

Comment:
In theory a former multibuild/multispec could also switch to pure multispec again

Reply:
I thought about it and considered it too academic taking the trend